src/util/numeric.cpp

```diff
@@ -188,14 +188,19 @@ u64 murmur_hash_64_ua(const void *key, int len, unsigned int seed)
 }
 
 /*
-	blockpos: position of block in block coordinates
+	blockpos_b: position of block in block coordinates
 	camera_pos: position of camera in nodes
 	camera_dir: an unit vector pointing to camera direction
 	range: viewing range
+	distance_ptr: return location for distance from the camera
 */
 bool isBlockInSight(v3s16 blockpos_b, v3f camera_pos, v3f camera_dir,
 		f32 camera_fov, f32 range, f32 *distance_ptr)
 {
+	// Maximum radius of a block.  The magic number is
+	// sqrt(3.0) / 2.0 in literal form.
+	const f32 block_max_radius = 0.866025403784 * MAP_BLOCKSIZE * BS;
```

Comment:
@Rogier-5  Just for future reference, some of us prefer that stuff like this is static const qualified rather than just const qualified. I doubt it makes a difference with modern compilers (easy to check) and it's more of a "style" issue, but static const is (from code I look at at, and obviously some others in #minetest-dev and other programming channels agree) is more idiomatic. It's a nitpick; not important really; and just "one of those things", which is why this was merged as-is (plus is consistent with code above anyway). Just thought I'd mention it for interests sake.